Resources/views/Collector/db.html.twig

```diff
@@ -40,9 +66,15 @@
                 <b>Cache puts</b>
                 <span class="sf-toolbar-status sf-toolbar-status-{{ collector.cachePutsCount > 0 ? 'yellow' : 'green' }}">{{ collector.cachePutsCount }}</span>
```

Comment:
you are keeping the green background here when the value is 0 while removing it for invalid entities. What is the reasonning about when a green background should be used in info pieces ?